### PR TITLE
update all.json

### PR DESCRIPTION
**Pull request by ical10**
|Site|Urlscan|
|--|--|
|claims.validations.gq|https://urlscan.io/result/8871271f-3346-4b0d-8540-a4b4ee16cc20/|
|coinationcryptoinsurance.us|https://urlscan.io/result/63ad4394-d34a-4b43-b4fd-b687d81c666b/dom/|
|coinsnationcryptoinsurance.com|https://urlscan.io/result/a669f429-d941-4318-8705-cd831248340f/dom/|
|connectrectification.org|https://urlscan.io/result/0882d3b5-2e8e-463b-a777-8a11fc91603a/dom/|
|cross-chaindapps.org|https://urlscan.io/result/6bea5690-5cf9-4dfc-a994-19e2016cd43c/dom/|
|dapp.myappsio.online|https://urlscan.io/result/f4030a94-d5f3-4213-8344-44115b92cf78/dom/|
|mainnet-connect.net|https://urlscan.io/result/66a97d12-a5ed-46d6-8874-ba7b90341a0d/dom/|
|mainnetconnections.com|https://urlscan.io/result/2dc9327f-d0c1-48cf-9279-5800ecc464a6/dom/|
|mainnetextension.net|https://urlscan.io/result/8884adcd-4106-4dc9-a15d-767a8cf08db9/|
|maintokenfix.site|https://urlscan.io/result/386220a6-3f18-4ba6-80a6-4c608d085ace/|
|mainvalidationsconnect.online|https://urlscan.io/result/5892e6dc-0de3-4831-8a57-7a902bd8f573/dom/|
|motionless-platform.000webhostapp.com|https://urlscan.io/result/0af58786-55a4-4d12-8710-c6a0d54d7103/|
|multi-entitychainfix.com|https://urlscan.io/result/3125956a-e61e-4035-a70b-7fe874fffe21/dom/|
|multicryptowallt.io|https://urlscan.io/result/ce3faee7-5413-423c-92c0-2481586d45cb/dom/|
|mymultinodesdefi.com|https://urlscan.io/result/80856054-b083-4a26-86ac-c7161adb7cbc/|
|serversync.netlify.app|(parity signer) https://urlscan.io/result/8ccbbc0d-d6c6-4db0-80b0-f710dfe8440e/dom/|
|smartdapvalidationsconnect.com|https://urlscan.io/result/ec296081-faf6-4d48-a032-edd5a516af06/dom/|
|walcon.us|https://urlscan.io/result/0f91d7ea-cd0a-4346-937d-f0f8baf47240/dom/|
|waletsdapp.net|https://urlscan.io/result/d62a2dda-0649-4179-8981-5f5559a116fd/dom/|
|walletaffixsynchronizer.com|https://urlscan.io/result/15ec351e-32ff-4e2b-b50c-439c565fba65/dom/|
|walletconnect.minehouse.tech|https://urlscan.io/result/9c8dd161-e67a-4e1a-b601-3e0bcb25bfe7/dom/|
|walletsvalidation.webflow.io|https://urlscan.io/result/2a833d5e-6f6f-47e5-a043-71f1abb0fb15/dom/|

claims validations gq_SITE
coinationcryptoinsurance us_SITE
coinsnationcryptoinsurance com_SITE
connectrectification org_SITE
cross-chaindapps org_SITE
dapp myappsio online_SITE
mainnet-connect net_SITE
mainnetconnections com_SITE
mainnetextension net_SITE
maintokenfix site_SITE
mainvalidationsconnect online_SITE
motionless-platform 000webhostapp com_SITE
multi-entitychainfix com_SITE
multicryptowallt io_SITE
mymultinodesdefi com_SITE
serversync netlify app_SITE
smartdapvalidationsconnect com_SITE
walcon us_SITE
walletaffixsynchronizer com_SITE
walletsvalidation webflow io_SITE

### CODE REVIEW

File: all.json [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
+		"multi-entitychainfix.com",
+		"mainvalidationsconnect.online",
+		"serversync.netlify.app",
+		"walletaffixsynchronizer.com",
+		"mainnetconnections.com",
+		"mainnet-connect.net",
+		"mainnetextension.net",
+		"coinsnationcryptoinsurance.com",
+		"coinationcryptoinsurance.us",
+		"dapp.myappsio.online",
```
Comment (jacogr) [inline all.json:21]:
```suggestion
		"dapp.myappsio.online",
		"myappsio.online",
```

File: all.json [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
+		"waletsdapp.net",
+		"claims.validations.gq",
```
Comment (jacogr) [inline all.json:23]:
```suggestion
		"claims.validations.gq",
		"validations.gq",
```

**Review by jacogr — COMMENTED** [review]
LGTM - just added some top-level domains as well. This just protects against them changing the subdomain.

(The first 2 top-levels are unreachable already, the last one not.)

**Review by frankywild — APPROVED** [review]

File: all.json [hunk 3]
```diff
@@ -1,6 +1,28 @@
 {
 	"allow": [],
 	"deny": [
+		"multicryptowallt.io",
+		"maintokenfix.site",
+		"walletsvalidation.webflow.io",
+		"connectrectification.org",
+		"motionless-platform.000webhostapp.com",
+		"mymultinodesdefi.com",
+		"walletconnect.minehouse.tech",
```
[the remark this answers is not in the document]
Comment (ical10) [reply all.json:10]: wait, isn't there some code in the CI which adds these top-level domains automatically? @jacogr

**Comment by ical10** [comment]
thanks Frank!

**Pull request merged**